### PR TITLE
Added Swift 3 storyboard templates for OSX

### PR DESCRIPTION
**Pull request by FelixII**
I've migrated the existing OSX storyboard templates to the new Swift 3 syntax.

### CODE REVIEW

**Comment by djbe** [comment]
Thank you for the PR. Is there a reason that both `default` and `lowercase` exist for swift3? I think only the lowercase one is needed.

On another (related) topic:
Can't we just unify the `osx` and `ios` templates into one like the other templates? I think that'd be a more worthwhile endeavour instead of adding more templates.

Couple of missing things:
- Unit tests for each template.
- Changelog entry.

**Comment by tbaranes** [comment]
> Can't we just unify the osx and ios templates into one like the other templates? I think that'd be a more worthwhile endeavour instead of adding more templates.

At the beginning, we searched a solution to have a common templates for all the gen files, but we concluded that storyboards will be an exception since the use case differ a bit from iOS whereas strings, colors, fonts are the same. You will find all the details to this in #101 and the PRs where the storyboards have been implemented.

**Comment by FelixII** [comment]
> Thank you for the PR. Is there a reason that both default and lowercase exist for swift3? I think only the lowercase one is needed.

You are absolutely right, thank you! The `lowercase` template does only affect the enum value, which is lowercase in Swift 3 by default. I will remove the superfluous template and supplement the missing items shortly.

**Comment by FelixII** [comment]
I've merged the two templates into one, added unit tests and a changelog entry. Please let me know if there is anything else missing or wrong.

**Comment by djbe** [comment]
Looks good, just a small changelog issue. Unless someone else (@AliSoftware?) has any input, I think we can merge this.

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ _None_
 
 ### New Features
 
+* Added a `storyboards-osx-swift3` template.
```
Comment (djbe) [inline CHANGELOG.md]: Please add 2 spaces at the end of the line (after the `.`).

Comment (AliSoftware) [inline CHANGELOG.md]: I've pushed the changes for this, will merge as soon as Travis is OK 👍

**Pull request merged**